airbyte-integrations/connectors/source-mysql/metadata.yaml

```diff
@@ -22,8 +22,8 @@ data:
   connectorIPCOptions:
     dataChannel:
       version: "0.0.1"
-      supportedSerialization: ["JSONL", "PROTOBUF"]
-      supportedTransport: ["SOCKET", "STDIO"]
+      supportedSerialization: ["JSONL"]
```

Comment:
Why this change? Also the version should be changed to 0.0.2

Reply:
Only testing clickhouse. 
I'll revert back